analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -106,15 +105,16 @@ internal fun SpdxDocument.projectPackage(): SpdxPackage? =
         ?.singleOrNull { it.packageFilename.isEmpty() || it.packageFilename == "." }
 
 /**
- * Get the [SpdxPackage] from the [SpdxExternalDocumentReference] that the [packageId] refers to, where [definitionFile]
- * is used to resolve local relative URIs to files.
+ * Get the [SpdxPackage] from the [SpdxExternalDocumentReference] using [loader] that the [packageId] refers to, where
```

Comment:
This makes the sentence hard to read, I'd rather add it to the beginning: "Use [load] to get..."